### DISCUSSION TITLE
bigtreetech_GD_TFT35_E3_V3.0

### DISCUSSION THREAD

**Discussion by Bapp1** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2628] [Q&A]
Installed my new tft screen today got it talking to the btt skr 3.0.1 motherboard with the marlin side all works as should but when i go into tft screen it almost appears to be in a demo mode i can touch buttons and move around the screens but it does nothing when trying to adjust fans or heat the bed up or nozzle etc. Ive tried downloading the latest BIGTREE_TFT35_V3.0_E3.27.x.bin file which matches my screen and put it on a fresh formatted sd card loaded that into the sc slot on the screen switched on and it says sd card inserted and under that is the btt logo, it takes seconds to go from that screen to the settings and connects to the printer.  Im at a loss please help someone tech savy as im not.

**Comment by oldman4U** [comment]
Read the Read Me at the front page of this repository

**Reply by Bapp1** [reply]
I did it didnt help as to why i asked for help but dont worry i sorted it another way so thanks for no help

**Comment by radek8** [comment]
Send photos connection and configuration files

**Comment by oldman4U** [comment]
He fixed it.

**Reply by radek8** [reply]
> He fixed it.

OK

**Comment by radek8** [comment]
BIGTREE_TFT35_V3.0_E3.27.x.bin file is not FW for your display. You write in the title that you have GD versions!

**Comment by oldman4U** [comment]
And he wrote that he read the read me and it did not help. As far as I know the difference between the versions is described there...🤪

**Comment by Bapp1** [comment]
Actually for your info apparently there was a coding error and unified are rolling out an update thats fixed the problems i was having and yes it may have been there oldman4u but to some of us github and how you get to the files or compile them is not as simple for us as it obviously is for you, i read the readme it didnt help im sorry i dont know how to find the files or how to use them once u do find them but hey obviously u dont like helping those that get stuck.

**Comment by oldman4U** [comment]
This is the discussion board of the BTT TFT Firmware repository and not a user group. User groups are there to help in case you get stuck with firmware, VSC and so on. Hope this helps

**Comment by radek8** [comment]
Guys, don't argue. 
@Bapp1 Still need help?

**Reply by Bapp1** [reply]
nope i got there in the end
